Extensions/TkSharp.Extensions.LibHac/Util/TkKeyUtils.cs

```diff
@@ -39,13 +39,16 @@ public static bool TryGetKeys(string? sdCardRootPath, [MaybeNullWhen(false)] out
         }
         
         string titleKeysFile = Path.Combine(target, "title.keys");
-        if (!File.Exists(titleKeysFile)) {
-            return null;
+        if (File.Exists(titleKeysFile)) {
+            KeySet keys = new();
+            ExternalKeyReader.ReadKeyFile(keys, titleKeysFilename: titleKeysFile, prodKeysFilename: keysFile);
+            return keys;
+        }
+        else {
+            KeySet keys = new();
+            ExternalKeyReader.ReadKeyFile(keys, prodKeysFilename: keysFile);
+            return keys;
         }
```

Comment:
Remove redundant else statement

```suggestion

        KeySet keys = new();
        ExternalKeyReader.ReadKeyFile(keys, prodKeysFilename: keysFile);
        return keys;
```